sos/report/plugins/snap.py

```diff
@@ -23,8 +24,42 @@ def setup(self):
         self.add_cmd_output("snap list --all", root_symlink="installed-snaps")
         self.add_cmd_output([
             "snap --version",
-            "snap changes"
+            "snap version",
+            "snap whoami",
+            "snap model --verbose",
+            "snap model --serial --verbose",
+            "snap services",
+            "snap connections",
+            "snap changes",
+            "snap changes --abs-time",
+            "snap validate",
+            "snap debug state --abs-time --changes /var/lib/snapd/state.json",
+            "snap debug stacktraces",
+            "snap get system -d",
         ])
+
+        all_pkgs = self.policy.package_manager.packages
+
+        for pkg_name in all_pkgs:
+            pkg = self.policy.package_manager.pkg_by_name(pkg_name)
+            if pkg['pkg_manager'] == 'snap':
+                self.add_cmd_output(f"snap connections {pkg['name']}")
+
         self.add_cmd_output("snap debug connectivity", timeout=10)
 
+        snap_changes = sos_get_command_output('snap changes')
+
+        if snap_changes['status'] == 0:
+            output = snap_changes['output']
+
+            for line in output.splitlines():
+                if line == "":
+                    continue
+                change = line.split()
+                if change[0] == "ID":
+                    continue
```

Comment:
Is this a header field, or something that can appear anywhere? If the former, let's just use `output.splitlines()[1:]` (or whatever number of header lines we need to skip).